vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -101,61 +100,62 @@ kubectl config current-context
 
-1. From the EKS cloud shell (or AWS client), login.
-2. Apply this version of NGINX:
+1. Create the NGINX ingress controller
 ```sh
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.3/deploy/static/provider/aws/deploy.yaml
 ```
-3. Get the ELB external URL for your NGINX install
+2. Get the ELB external URL for your NGINX install
 ```sh
 kubectl get services -n ingress-nginx
 ```
 
-Get the value of the external address (EXTERNAL-IP) for the ingress-nginx-controller:
+For example:
 ```
 NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP                        
 ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1ff4dadb44639a6bbf707e-0e0a483d966405ee.elb.us-east-2.amazonaws.com
 ```
-You'll use this address for YOUR-HOST-NAME in the steps below.
+The value of the external URL (EXTERNAL-IP) should be available within a few minutes. You'll use this address for `YOUR-HOST-NAME` in the steps below.
 
 ::: tip
-NGINX is working correctly if a `404 Not Found` error page is generated when accessing the EXTERNAL-IP from your browser. Alternatively, you can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-nginx-configuration) to optimize the configuration for Entando.
+NGINX is working correctly if a `404 Not Found` error page is generated when accessing the `YOUR-HOST-NAME` from your browser. For a more complete test, you can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-nginx-configuration) to optimize the configuration for Entando.
 :::
 
-See the [NGINX Ingress Controller Installation Guide](https://kubernetes.github.io/ingress-nginx/deploy/#aws) and the [NGINX as Ingress Controller on EKS](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more details.
+See the [NGINX AWS Guide](https://kubernetes.github.io/ingress-nginx/deploy/#aws) and [NGINX EKS Guide](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more details.
 
 ### Install the Entando Custom Resource Definitions
-Once per cluster, deploy the `Entando Custom Resources`.
-1. Download the Custom Resource definitions and deploy the cluster scoped resources:
-```
+
+1. Apply the cluster-scoped Custom Resource Definitions (CRDs)
+```sh
 kubectl apply -f https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/namespace-scoped-deployment/cluster-resources.yaml
 ```
-2. Create the namespace for the Entando Application:
+This is required only once per cluster.
```

Comment:
I would put this above with the first part of step 1. Because the list is long, it reads easier for me when all the instructions are in the same expected place